lib/db.js

```diff
@@ -27,6 +31,11 @@ module.exports = (
 
   function DB(options) {
     this.pool = new Pool(options.url)
+    this.redis = config.redis.enabled ? redis.createClient({
+      host: config.redis.host,
+      port: config.redis.port,
+      prefix: config.redis.sessionsKeyPrefix
+    }) : undefined
```

Comment:
@udaraweerasinghege can we add a `log.info` after this to show us if redis got enabled or not. 

If it is `on` then log the config values it is using, so we can see this info when server starts